sdk/communication/Azure.Communication.MediaComposition/swagger/readme.md

```diff
@@ -0,0 +1,54 @@
+# communicationservices
```

Comment:
Instead of copying everything from the swagger folder,  you can just link the raw file from your PR. This is what we did in the past when the swagger has not been checked in yet. 

E.g. https://raw.githubusercontent.com/Azure/azure-rest-api-specs/85fd44e61de4bb5d4d18933ecf72ae2bcc657f45/specification/communication/data-plane/MediaComposition/readme.md and remove all the local `swagger/` files